### ISSUE TITLE
Uncaught Error: Cannot find module './.ctags'

### ISSUE THREAD

**Issue by xinfli** [atom/symbols-view#71]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.165.0
**System**: Microsoft Windows 7 Enterprise 
**Thrown From**: [symbols-view](https://github.com/atom/symbols-view) package, v0.70.0

### Stack Trace

Uncaught Error: Cannot find module './.ctags'

```
At module.js:346

Error: Cannot find module './.ctags'
  at Module._resolveFilename (module.js:344:15)
  at Function.Module._resolveFilename (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\src\module-cache.js:380:52)
  at Function.require.resolve (module.js:396:19)
  at TagGenerator.module.exports.TagGenerator.generate (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\node_modules\symbols-view\lib\tag-generator.js:89:34)
  at FileView.module.exports.FileView.generateTags (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\node_modules\symbols-view\lib\file-view.js:100:62)
  at FileView.module.exports.FileView.populate (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\node_modules\symbols-view\lib\file-view.js:95:21)
  at FileView.module.exports.FileView.toggle (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\node_modules\symbols-view\lib\file-view.js:72:14)
  at atom-text-editor.editorSubscription.atom.commands.add.symbols-view:toggle-file-symbols (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\node_modules\symbols-view\lib\main.js:21:43)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\src\command-registry.js:243:29)
  at c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\src\command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\node_modules\atom-keymap\lib\keymap-manager.js:549:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\node_modules\atom-keymap\lib\keymap-manager.js:391:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (c:\Users\xinfengl\AppData\Local\atom\app-0.165.0\resources\app\src\window-event-handler.js:167:20)

```

### Commands

```
     -0:03.3 core:copy (input.hidden-input)
     -0:00.0 symbols-view:toggle-file-symbols (input.hidden-input)
```

### Config

```
{}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

/cc @atom/core

**Comment by kevinsawicki** [comment]
This is a duplicate of #63, it is fixed in the latest version of Atom (0.177). If you upgrade versions it should go away.

**Comment by mrkhanakia** [comment]
Hi
There can 2 reason that you are getting above error
1. ctags module is not installed
2. if ctags module installed correctly then NODE_PATH is not set which is causing error.
So i have Centos server and my shared node_modules are installed in that directory /usr/local/lib/node_modules

So i did SSH in my system as normal user (Not root user) and then did fire the following command
export NODE_PATH=/usr/local/lib/node_modules

After that problem SOLVED
